Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -21,6 +21,19 @@ compatible with workspaces is described in the
 TCA reference (in the :ref:`description of the "ctrl" section <t3tca:ctrl>`
 and in the :ref:`description of the "versioningWS" property <t3tca:ctrl-reference-versioningws>`).
 
+You might want to turn the workspace off for certain tables. 
+You can do so in the :file:`Configuration\TCA\Overrides\example_table.php`::
+
+   $GLOBALS['TCA']['example_table']['ctrl']['versioningWS'] = false;
+
+See :ref:`t3sitepackage:start` and :ref:`storing-changes-extension-overrides` .
+This will lead to all t3ver_* fields of the example table to be marked as obsolete,
+if they have not be defined explicitly in the extension. A subsequent DB schema update will then drop these fields.
+
+.. note::
```

Comment:
I'd rather drop this note since it leads to more confusion than it helps. The `AdditionalConfiguration.php` is not mentioned anywhere in this manual.

Reply:
But this information is a warning, that the formerly stored workspace records will get lost with the next DB update. This could help some persons who maybe want to keep this data for a while. And maybe I am not the only one person who did think, that `AdditionalConfiguration.php` could be used for this.

Reply:
Notice that I only referred to the note about the `AdditionalConfiguration.php`. Since you already mention and showcase TCA overrides there is not much value in saying "doing it wrong won't work". ;-)

Reply:
I have removed the hint about `AdditionalConfiguration.php` and changed the previous part. Nobody should search for the other wrong way.